components/nginx/config/https-file-server.cloudify

```diff
@@ -2,15 +2,15 @@
 server {
     # server listening for internal requests
     listen              53229 ssl;
-    server_name         _;
+    server_name         127.0.0.1;
 
     # force http redirect to https
     if ($scheme = http) {
         return 301 https://$server_name$request_uri;
     }
 
-    ssl_certificate     {{ ctx.target.instance.runtime_properties.external_cert_path }};
-    ssl_certificate_key {{ ctx.target.instance.runtime_properties.external_key_path }};
+    ssl_certificate     {{ ctx.source.instance.runtime_properties.internal_cert_path }};
```

Comment:
Wait, what am I missing? You *don't* need the key in nginx? In every example I saw both cert and key were provided.